go.mod

```diff
@@ -45,6 +45,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require golang.org/x/net v0.0.0-20210825183410-e898025ed96a
```

Comment:
Out of curiosity: Do you know what triggered this change?

Reply:
Nope.
This was reported by verify-modules, I fixed it, but not investigated it deeply